mod/metrics/set/type_plus_right/metric_value/checked_by.rb

```diff
@@ -84,71 +98,68 @@ def option_label_text _option_name
     "#{request_icon} Request that another researcher double check this value"
   end
 
-  view :core, cache: :never do
-    unless card.check_requested? || card.checked? || card.allowed_to_check?
-      return ""
-    end
-    wrap_with :div do
-      [
-        wrap_with(:h5, "Review"),
-        wrap_with(:p, "Does the value accurately represent its source?"),
-        check_interaction
-      ]
-    end
+  view :core, template: :haml
+
+  # FIXME: this view is wrongly cached if it's moved to a haml template
+  #   To see how it fails add `template: :haml` and run the double_check.feature
+  view :check_interaction, cache: :never do
+    return unless allowed_to_check?
+    "<p>Does the value accurately represent its source?</p>" +
+      if user_checked?
+        check_button "Uncheck", action: :uncheck
+      else
+        check_button("Yes, I checked", action: :check) + fix_link
+      end
+  end
+
+  def check_interaction_wrapper
+    return unless allowed_to_check?
+    "Does the value accurately represent its source?" +
+      if user_checked?
+        check_button "Uncheck", action: :uncheck
+      else
+        check_button("Yes, I checked", action: :check) + fix_link
+      end
   end
 
   view :icon, cache: :never do |args|
-    if card.checked?
+    if checked?
       double_check_icon args
-    elsif card.check_requested?
+    elsif check_requested?
       request_icon args
     else
       ""
     end
   end
 
-  def check_interaction
-    if card.user_checked?
-      user_checked_text
-    elsif card.checked?
-      _render_checked_by_list
+  def verb
+    answer.editor_id ? "last updated" : "created"
+  end
+
+  def checkers_list
+    checkers.map { |n| nest n, view: :link }.to_sentence
+  end
+
+  def short_checkers_list checker_view=:link
```

Comment:
as I mentioned on the story, I think it's ok just to put in all checkers for now.  more than 3 won't happen much.  (this method fails codeclimate and I think is uncalled?). can just comment out for now...